app.css

```diff
@@ -7,7 +7,7 @@
 body {
   background-color: #1c1a19;
   width: 375px;
```

Comment:
- [OPTIONAL] I suggest using %,vh and vw as much as possible to make your project responsive :recycle: